scripts/build.ts

```diff
@@ -153,13 +155,15 @@ const buildSimpleIconsSvgFontFile = async (style) => {
   return { unicodeHexBySlug, svgFileContent };
 };
 
-const buildSimpleIconsCssFile = (unicodeHexBySlug) =>
+const buildSimpleIconsCssFile = (
+  unicodeHexBySlug: UnicodeHexBySlug,
+): Promise<string> =>
   new Promise(async (resolve, reject) => {
     try {
-      let cssFileContent = await fs.readFile(CSS_BASE_FILE);
+      let cssFileContent = await fs.readFile(CSS_BASE_FILE, 'utf8');
```

Comment:
There is a `UTF8` global in this script.

```suggestion
      let cssFileContent = await fs.readFile(CSS_BASE_FILE, UTF8);
```